src/api.ts

```diff
@@ -140,6 +140,12 @@ class RippleAPI extends EventEmitter {
         this.emit('connected')
       })
       this.connection.on('disconnected', code => {
+        // This is a backwards-compatible fix for this change in the ws library: 
+        //   https://github.com/websockets/ws/issues/1257
+        // TODO: Remove in next major, breaking version
+        if (code === 1005) {
```

Comment:
Don't mutate parameter inputs. Create a new variable, `normalizedCode`. This helps prevent bugs when future programmers don't realize `code` was mutated. 

I know this example is super simple but it's best practice.

----

Optionally, you can do this in a one liner:
```javascript
const nomralizedCode = code !== 1005 ? code : 1000
```

Reply:
Big +1 to the sentiment. I was balancing that against the desire to make sure that this would be easy to remove in the next major version (deleting lines vs. refactoring lines). 

Given that this is pretty self contained, do you still feel strongly? Happy to make the change if so, just wanted to give the missing context :)

Reply:
I do feel strongly - I've watched people get burned on this too many times.

Reply:
👍 sgtm, made the change